addons/account/models/account.py

```diff
@@ -677,9 +677,10 @@ def _get_bank_statements_available_sources(self):
              "Batch Deposit: Encase several customer checks at once by generating a batch deposit to submit to your bank. When encoding the bank statement in Odoo,you are suggested to reconcile the transaction with the batch deposit. Enable this option from the settings.")
     outbound_payment_method_ids = fields.Many2many('account.payment.method', 'account_journal_outbound_payment_method_rel', 'journal_id', 'outbound_payment_method',
         domain=[('payment_type', '=', 'outbound')], string='For Outgoing Payments', default=lambda self: self._default_outbound_payment_methods(),
-        help="Manual:Pay bill by cash or any other method outside of Odoo.\n"\
+        help="Manual:Pay bill by cash, credit card or any other method outside of Odoo.\n"\
```

Comment:
Do you mean bank? You don't get to decide to pay your vendor by credit card if he doesn't have a processor.

Reply:
Yeah, it's incorrect. I'm not sure why I made this change. I have removed it entirely.